exampleFolder/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+var app = express();
+const axios = require('axios');
+const { response } = require('express');
+
+//POST Route (/addPerson)
+app.post('/addPerson', (req, res) => {
+    console.log(req.body)
+    const body = req.body
+
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson',body)
+    .then((Response) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
+    })
+
+    .catch((error) => {
+        console.log("Error Has Occured")
+        res.json({ErrorResponse: "Unsuccessful Route"})
+    })
+});
+
+//DELETE Route (/deletePerson/:id)
+app.delete('/deletePerson/:id', (req, res) => {
+    const { id } = req.params;
+
+    axios.delete('https://java-sample-api-2020.herokuapp.com/deletePerson?id='+id, id)
```

Comment:
Here on line 27. At the end of the app.delete parameters, the extra 'id' after the comma is not needed. Only the url concantenated with id.

Reply:
I think Jonathan is right here. One thing I would suggest is to understand why this is the case though. What is it that you are doing here and why should you not be doing it here.

Reply:
> Here on line 27. At the end of the app.delete parameters, the extra 'id' after the comma is not needed. Only the url concantenated with id.

Fixed.